src/utils/StringUtils.js

```diff
@@ -113,6 +113,16 @@ define(function (require, exports, module) {
     }
     
     /**
+     * Returns true if the given string starts with the given prefix.
+     * @param   {String} str
+     * @param   {String} prefix
+     * @returns {Boolean}
+     */
+    function startsWith(str, prefix) {
+        return str.slice(0, prefix.length) === prefix;
+    }
```

Comment:
Lodash now offers [startsWith](https://github.com/lodash/lodash/wiki/Changelog#notable-changes) from 3.0 onward. So may be we could make this private to `PerfUtils`, so we don't have to give deprecation warnings when the new lodash update is merged.